pom.xml

```diff
@@ -170,9 +170,8 @@
 
       <!-- Conflict with AssertJ -->
       <dependency>
-        <groupId>net.bytebuddy</groupId>
-        <artifactId>byte-buddy</artifactId>
-        <version>1.14.18</version>
+        <groupId>io.jenkins.plugins</groupId>
```

Comment:
TBH I don't even know why it's here. maybe by reading at the comment, some ancient upperbound issue.
I would simply remove this declaration which is in dptMngt section and not in dependencies section (changing to the api wouldn;t have much effect as it;s not even packaged with the hpi)